control/responsible_role.go

```diff
@@ -1,15 +1,16 @@
 package control
 
 import (
-	"github.com/jbowtie/gokogiri/xml"
 	"errors"
+	"fmt"
 	"regexp"
 	"strings"
-	"fmt"
+
+	"github.com/jbowtie/gokogiri/xml"
```

Comment:
lol!! No worries. `go-plus` is running `go fmt` after every save. side note:  we probably need to activate this for add this to our codeclimate rules: https://github.com/codeclimate-community/codeclimate-gofmt